launch/test/launch/utilities/test_type_utils.py

```diff
@@ -0,0 +1,149 @@
+# Copyright 2020 Open Source Robotics Foundation, Inc.
```

Comment:
@ivanpauno this is missing some tests e.g. for `normalize_typed_substitution()` and `perform_typed_substitution()`.

Reply:
this is missing a lot of tests :joy:

Reply:
Ok, I think those functions are well covered now.